docs/release-notes/v1.50-v1.59.md

```diff
@@ -10,6 +10,224 @@
+
+![variantFeature.png](1.59/variantFeature.png)
+<figcaption>The feature search capability using <strong>variantFeature</strong> is added, aiding targeted gene searches</figcaption>
+
+#### [Curators]
+
+* **New endpoints for importing, and retrieving Reference Genomes**
+
+Users can now import different versions and builds of reference genomes. This feature allows users to significantly enhance flexibility, accuracy, and relevance in variant analysis.
+
+Uploading reference genomes was already supported in ODM but only via .py script. From this release (1.59), uploading of reference genomes is now available via a new API endpoint: <code>POST api/v1/reference-genomes</code>, part of the new definition **referenceGenome**.
+
+* ??? abstract "Important considerations of the Reference Genome endpoints"
+    * Only users that are part of the Curator Group can execute the endpoint api/v1/reference-genomes to upload and query reference genomes
+    * Each uploaded reference genome should be unique according to a combination of mandatory fields specified in a request body
+    * Reference genome requests should include the following parameters:
+
+    * | Field Name | Required | Description |
+      | :---- | :---- | :---- |
+      | annotationUrl | No | link to the non-vertebrae organism or any custom reference genome in .gtf format, archived in \`.gz\` format. Note: if any of the parameters below do not correspond to the available Ensembl data (for example, organism name or assembly is changed), the link must be provided. |
+      | species | Yes | Scientific name with the first uppercase letter and underscore as a delimiter (e.g. Homo\_sapiens).  |
+      | assembly | Yes | Major version (e.g. \`GRCh38\`) |
+      | release | Yes | Ensembl reference genome minor version (for example, \`75\` or \`109\`) |
+      | name | No | Customized reference genome title that is used for linkage variant files. By default is combined from species, assembly, and release parameters: \`{species} reference genome {assembly}.{release}\` |
+
+* ??? danger "New Endpoints to retrieve and import Reference Genomes"
+    * GET /api/v1/reference-genomes
+    * POST /api/v1/reference-genomes
+    * GET /api/v1/reference-genomes/{id}
+
+    ![RefGenome.png](1.59/RefGenome.png)
+    <figcaption>The definition <strong>referenceGenome</strong> has been created. This definition includes endpoints to retrieve and upload reference genomes</figcaption>
+
+* **Improved Support for Files (formerly known as Attachments): Add POST (job) endpoint to upload Files**
+
+On this release, the endpoint <code>POST /api/v1/jobs/import/file</code>, which allows the importing of Files (attachments) via API, has been incorporated to import file jobs efficiently.
+
+* ??? abstract "Important considerations"
+    * Upon calling the endpoint
+        * An attachment object is created in ODM.
+        * The reference between the specified study and the created object is created
+    * Request body:
```

Comment:
Please update request body it lacks two parameters: `metadataLink` and `dataClass` (which is mandatory)